griptape/tasks/text_to_image_task.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import annotations
+
+from typing import Callable
+
+from attr import define
+
+from griptape.artifacts import ImageArtifact, TextArtifact
+from griptape.tasks import BaseImageGenerationTask
+from griptape.utils import J2
+
+
+@define
+class TextToImageTask(BaseImageGenerationTask):
```

Comment:
This feels like it should be inheriting from `BaseTextInputTask`, but it gets kind of messy to do so :/

Reply:
It does! In response to the previous review round, BaseImageGenerationTask inherits from BaseTextInputTask to avoid repition of ruleset and input logic.